tests/upgrade/post/go.mod

```diff
@@ -17,7 +17,7 @@ replace (
 
 require (
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/oasisprotocol/oasis-core/go v0.2202.1-0.20230905072806-6925b49b145a
```

Comment:
The 8th line should be changed to
```
	github.com/cometbft/cometbft => github.com/oasisprotocol/cometbft v0.37.2-oasis1
```
indirect dependencies removed and `go mod tidy` should be ran again. That will produce the following diff.

```
+       github.com/btcsuite/btcd v0.20.1-beta // indirect
-       github.com/btcsuite/btcd/chaincfg/chainhash v1.0.2 // indirect
-       github.com/mitchellh/go-testing-interface v1.0.0 // indirect
+       github.com/mitchellh/go-testing-interface v0.0.0-20171004221916-a61a99592b77 // indirect
```